roles/openshift_master_facts/test/openshift_master_facts_bad_input_tests.py

```diff
@@ -0,0 +1,57 @@
+import copy
+import os
+import sys
+
+from ansible.errors import AnsibleError
+import pytest
+
```

Comment:
Any objections to adding `pytestmark = pytest.mark.unit` to allow for filtering out just the unit tests?

That would also allow to use marks for specifying functional tests, slow tests, etc to filter the tests that are run.